cypher-shell/src/main/java/org/neo4j/shell/commands/Param.java

```diff
@@ -14,13 +14,13 @@
 /**
  * This command sets a variable to a name, for use as query parameter.
  */
-public class Set implements Command {
+public class Param implements Command {
```

Comment:
We should consider making the change to the Test as well, it is still called SetTest.java